lib/iptest.js

```diff
@@ -1,12 +1,17 @@
-module.exports =(function(){
-	var testIp = function( ipAddress ){
-		var sds = '192.168.208', valid = true;
-		//Comment below 3 for development
-		if( ipAddress.substr(0,11) != sds){
-			valid = false;
-		}
-		return valid;
-	};
+var json = require('../public/config.json');
 
-	return {testIP : testIp};
-})();
\ No newline at end of file
+module.exports = (function() {
+    var testIp = function(ipAddress) {
+        var sds = '192.168.208',
+            valid = true;
+        //Comment below 3 for development
```

Comment:
You should remove this line now. Also add the description about env config variable to readme.